ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,414 @@
+    {
+      id: 'Cloud Chaotic Grim Embrace Collector',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E39', '9E3A'], source: 'Cloud of Darkness', capture: true },
+      run: (data, matches) => {
+        data.deadlyEmbraceDodgeDirectionCollected = matches.id === '9E39' ? 'back' : 'front';
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deadly Embrace',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1055', capture: true },
+      condition: Conditions.targetIsYou(),
+      preRun: (data) =>
+        data.deadlyEmbraceDodgeDirection = data.deadlyEmbraceDodgeDirectionCollected,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 7,
+      durationSeconds: 7,
+      infoText: (data, _matches, output) => output[data.deadlyEmbraceDodgeDirection]!(),
+      outputStrings: {
+        back: {
+          en: 'Bait hand, dodge backwards',
+        },
+        front: {
+          en: 'Bait hand, dodge forwards',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Rapid-sequence Particle Beam',
+      type: 'StartsUsing',
+      netRegex: { id: '9E40', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Wild Charge (alliances)',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Unholy Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: 'A12D', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.healerGroups!(),
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Flare Marker',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.flare!(),
+      outputStrings: {
+        flare: {
+          en: 'Flare on you',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Break IV',
+      type: 'StartsUsing',
+      // sometimes listed as coming from Death's Hand due to actor re-use,
+      // source omitted to prevent trigger issues.
+      netRegex: { id: '9E50', capture: false },
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Cloud Chaotic Endeath IV Collector',
+      type: 'StartsUsing',
+      netRegex: { id: '9E53', source: 'Cloud of Darkness', capture: false },
+      run: (data) => data.bladeOfDarknessFollowup = 'death',
+    },
+    {
+      id: 'Cloud Chaotic Enaero IV Collector',
+      type: 'StartsUsing',
+      netRegex: { id: '9E54', source: 'Cloud of Darkness', capture: false },
+      run: (data) => data.bladeOfDarknessFollowup = 'aero',
+    },
+    {
+      id: 'Cloud Chaotic Death IV',
+      type: 'StartsUsing',
+      netRegex: { id: '9E43', source: 'Cloud of Darkness', capture: false },
+      response: Responses.getOutThenIn(),
+    },
```

Comment:
I don't think `getOutThenIn` is a good match here because you have to go really far out, or you have to KB prevent to not get sucked in, but I don't really have better text for this one.

Reply:
> I don't think `getOutThenIn` is a good match here because you have to go really far out, or you have to KB prevent to not get sucked in, but I don't really have better text for this one.

How about `Far away => in`?